plugins/vagrant/vagrant.plugin.sh

```diff
@@ -55,22 +55,24 @@ function vagrant-status() {
 
 function vagrant-ssh() {
     local VMCOUNT
-    VMCOUNT="$(vagrant status | grep -c running)"
+    VMCOUNT=$(vagrant status | grep -c running)
     local VMDEFAULT
-    VMDEFAULT="$(vagrant status | grep -w default | grep -c running)"
+    VMDEFAULT=$(vagrant status | grep -w default | grep -c running)
```

Comment:
Note: We actually do not need to quote them in the right-hand side of the variable assignments (unless we want to include literal shell special characters) because the pathname expansions and word splitting does not occur. An exception is a case that the right-hand side contains `$*` or `${arr[*]}` with which some older versions of Bash have bugs.

Reply:
> Note: We actually do not need to quote them in the right-hand side of the variable assignments (unless we want to include literal shell special characters) because the pathname expansions and word splitting does not occur. An exception is a case that the right-hand side contains `$*` or `${arr[*]}` with which some older versions of Bash have bugs.

Okay, thank you by the advice.

Reply:
Yeah, this is just a matter of style after all. You can use either style in your scripts. BTW, `shellcheck` (of at least recent versions) doesn't warn about this. Anyway, thank you for your contribution!